package.json

```diff
@@ -12,13 +12,13 @@
     }
   },
   "scripts": {
-    "build": "cross-env NODE_ENV=production tsc && vite build",
+    "build": "cross-env NODE_ENV=production tsc --noEmit && vite build",
     "build:watch": "cross-env NODE_ENV=production vite build --watch",
     "build-dev": "cross-env NODE_ENV=development tsc && vite build",
     "clean": "cross-env rimraf dist coverage lib",
-    "coverage": "cross-env npm run build && jest --coverage",
+    "coverage": "cross-env npm run build && vitest --coverage.enabled",
```

Comment:
why this ,enabled?
in the frontend, we don't need this, vitest run --coverage is enough